jumpscale/servers/threebot/threebot.py

```diff
@@ -273,6 +281,51 @@ def delete(self, package_name):
         self.packages.pop(package_name)
         self.save()
 
+    def install_package(self, package_name):
```

Comment:
what's the difference between install and install package ?
should be more descriptive name